index.js

```diff
@@ -179,8 +179,8 @@ module.exports = class Payment extends EventEmitter {
   }
 
   static supports (payment) {
-    const supportedCurrencies = ['LightningBTC', 'LightningSats']
-    return supportedCurrencies.includes(payment.currency)
+    const supported = ['BTC', 'SATS']
+    return payment.method === 'Lightning' && supported.includes(payment.currency.toUpperCase())
```

Comment:
can you also do a case-insensitive compare of the method? (just toLowerCase() both slides)

Reply:
also we need to guard .currency here (ie return !!payment.currency && (... the rest)